Sources/Application/View/View.swift

```diff
@@ -5,6 +5,19 @@ public class View: Viewable {
 		self.io = io
 	}
 
+	public func promptMaxNumberOfGuesses() {
+		io.display("How many guesses do you want to have for this game?")
+	}
+
+	public func receiveMaxNumberOfGuesses() -> Int {
+		var maxNumberOfGuesses = io.getUserInput()
+		while (maxNumberOfGuesses == "" || !(validNumber(maxNumberOfGuesses))) {
+			io.display("Invalid number. Please enter a number greater than 0")
```

Comment:
This was addressed. In the `ViewSpec` I am checking if the `MockIO` display method is being called. The method check is being conducted in the `GameSpec` to verify that the method is called and that the return value is correct.